activities/Vote.activity/js/activity.js

```diff
@@ -640,168 +640,165 @@ var app = new Vue({
+						position: "bottom",
 						title: this.l10n.stringTutoSettingsQuestionTitle,
-						content: this.l10n.stringTutoSettingsQuestionContent
+						intro: this.l10n.stringTutoSettingsQuestionContent
 					},
 					{
 						element: "#type",
-						placement: "bottom",
+						position: "bottom",
 						title: this.l10n.stringTutoSettingsTypeTitle,
-						content: this.l10n.stringTutoSettingsTypeContent
+						intro: this.l10n.stringTutoSettingsTypeContent
 					},
 					{
 						element: ".add-option-button",
-						placement: "top",
+						position: "top",
 						title: this.l10n.stringTutoSettingsAddOptionButtonTitle,
-						content: this.l10n.stringTutoSettingsAddOptionButtonContent
+						intro: this.l10n.stringTutoSettingsAddOptionButtonContent
 					},
 					{
 						element: ".option#0",
-						placement: "top",
+						position: "top",
 						title: this.l10n.stringTutoSettingsOptionTitle,
-						content: this.l10n.stringTutoSettingsOptionContent
+						intro: this.l10n.stringTutoSettingsOptionContent
 					},
 				]);
 			} else {
 				switch(this.currentView) {
 					case 'polls-grid':
 						steps = steps.concat([
 							{
-								element: "",
-								orphan: true,
-								placement: "bottom",
 								title: this.l10n.stringTutoExplainTitle,
-								content: this.l10n.stringTutoExplainContent
+								intro: this.l10n.stringTutoExplainContent
 							},
 							{
-								element: ".poll-card#0",
-								placement: "right",
+								element: ".poll-card [id='#0']",
+								position: "right",
 								title: this.l10n.stringTutoPollCardTitle,
-								content: this.l10n.stringTutoPollCardContent
+								intro: this.l10n.stringTutoPollCardContent
 							},
 							{
 								element: ".search-input",
-								placement: "bottom",
+								position: "bottom",
 								title: this.l10n.stringTutoPollSearchTitle,
-								content: this.l10n.stringTutoPollSearchContent
+								intro: this.l10n.stringTutoPollSearchContent
 							},
 							{
 								element: "#settings-button",
-								placement: "bottom",
+								position: "bottom",
 								title: this.l10n.stringTutoSettingsButtonTitle,
-								content: this.l10n.stringTutoSettingsButtonContent
+								intro: this.l10n.stringTutoSettingsButtonContent
 							},
 							{
 								element: "#history-button",
-								placement: "bottom",
+								position: "bottom",
 								title: this.l10n.stringTutoHistoryButtonTitle,
-								content: this.l10n.stringTutoHistoryButtonContent
+								intro: this.l10n.stringTutoHistoryButtonContent
 							},
 							{
 								element: "#network-button",
-								placement: "bottom",
+								position: "bottom",
 								title: this.l10n.stringTutoNetworkButtonTitle,
-								content: this.l10n.stringTutoNetworkButtonContent
+								intro: this.l10n.stringTutoNetworkButtonContent
 							},
 						]);
 						break;
 					case 'history':
 						steps = steps.concat([
 							{
 								element: ".history-item#0",
```

Comment:
Update the selector here as well.

Reply:
I have made changes but please can you tell why tutorial was working fine without any change here in line 697 & 746.
But it was giving error when unchanged on line 643.

Reply:
This is because these steps on line number 697 and 764 are added in tutorial when you click on settings button or history button. If you read the code carefully you'll notice [here](https://github.com/llaske/sugarizer/pull/1106/files#diff-a2f3a7b6faa46b3bb7662cf88fc14ba88bb45db6d70d4497f3631d0abde08715L640) we are checking if this.setings is true, then we are adding a different set of steps and in else part we are using switch case where we add steps based on different cases.
Hope it helps!

Reply:
Thanks @disha1202 .